src/components/Main/NavCircle/NavCircle.tsx

```diff
@@ -1,19 +1,173 @@
 import React from 'react';
+import styled from 'styled-components';
 
-// TODO: Add circle background
-// TODO: Add three links
-// TODO: Add social media links
+// TODO: Add circle background (done)
+// TODO: Add three links 
+// TODO: Add social media links (done)
 // TODO: (if needed in a future PR) depending on scroll position, change the color of circle border
-// TODO: Make absolute on the page so it follows the screen
+// TODO: Make absolute on the page so it follows the screen (done)
+
+const StyledLinkOne = styled.a`
```

Comment:
sorry, not sure if i made the announcement, but last meeting we decided to stay off styled components unless absolutely necessary. try instead using classes for as much as possible